global.json

```diff
@@ -1,7 +1,6 @@
 {
   "sdk": {
     "rollForward": "latestFeature",
-    "version": "6.0.100",
-    "allowPrerelease": "true"
+    "version": "7.0.100-preview"
```

Comment:
we need exact version here?
https://docs.microsoft.com/en-us/dotnet/core/tools/global-json#version

Reply:
I tried both. Seems to work fine with just "-preview" so I went with the less restrictive version.